src/components/spreadsheet/spreadsheet.ts

```diff
@@ -166,8 +163,8 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
       "CTRL+F": () => this.toggleSidePanel("FindAndReplace", {}),
     };
     useSubEnv({
-      model: this.model,
-      isDashboard: () => this.model.getters.isDashboard(),
+      model: this.props.model,
```

Comment:
I think I would have introduced a getter to avoid coupling the code in the component to the props (and avoid the diff).
But it's ok